packages/compass-e2e-tests/tests/collection-bulk-update.test.ts

```diff
@@ -50,6 +50,10 @@ describe('Bulk Update', () => {
 
     // Check the telemetry
     const openedEvent = await telemetryEntry('Bulk Update Opened');
```

Comment:
```
const {connection_id, ...openedEvent} = await telemetryEntry('Bulk Update Opened');

expect(connection_id).to.exist;
expect(openedEvent).to.deep.equal({
      isUpdatePreviewSupported: true,
    });
```

Reply:
so I checked the `browser.listenForTelemetryEvents` and it is doing a copy of the properties!